sql/moz-fx-data-shared-prod/telemetry/active_users/view.sql

```diff
@@ -0,0 +1,189 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.active_users`
+AS
+-- Distribution_id is only collected for Desktop and Fenix as of Q1/2024.
+-- See e.g. Firefox iOs https://dictionary.telemetry.mozilla.org/apps/firefox_ios?page=1&search=distribution_id
+-- TODO: Integrate distribution_id in fenix.firefox_android_clients & backfill to use it here.
+WITH fenix_distribution_id AS (
+  SELECT
+    client_info.client_id AS client_id,
+    sample_id AS sample_id,
+    DATE(submission_timestamp) AS submission_date,
+    ARRAY_AGG(
+      metrics.string.metrics_distribution_id IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS distribution_id,
+  FROM
+    `moz-fx-data-shared-prod.fenix.baseline`
+  GROUP BY
+    client_id,
+    sample_id,
+    submission_date
+)
+-- Firefox Desktop
+SELECT
+  submission_date,
+  client_id,
+  CASE
+    WHEN isp_name = 'BrowserStack'
+      THEN CONCAT(app_name, ' ', isp_name)
+    WHEN distribution_id = 'MozillaOnline'
+      THEN CONCAT(app_name, ' ', distribution_id)
+    ELSE app_name
+  END AS app_name,
+  days_seen_bits,
+  days_active_bits,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) = 0 AS BOOLEAN) AS is_dau,
+  CAST(mozfun.bits28.days_since_seen(days_active_bits) <= 7 AS BOOLEAN) AS is_wau,
```

Comment:
The `active_users_aggregates` table currently uses `<` (not `<=`). Since this is `days_since_seen`, I think it's zero-indexed, so we need to use `<`.

https://github.com/mozilla/bigquery-etl/blob/366ec3550855a05fc716a1c9562823a359bf94a4/sql_generators/active_users_aggregates_v3/templates/desktop_query.sql#L77

Reply:
Indeed!